pkgs/development/python-modules/python-lsp-black/default.nix

```
@@ -0,0 +1,34 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, black
+, toml
+, pytestCheckHook
+, python-lsp-server
+, isPy3k
+}:
+
+buildPythonPackage rec {
+  pname = "python-lsp-black";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "python-lsp";
+    repo = "python-lsp-black";
+    rev = "v${version}";
+    sha256 = "1blxhj70jxb9xfbd4dxqikd262n6dn9dw5qhyml5yvdwxbv0bybc";
+  };
+
+  disabled = !isPy3k;
```

```suggestion
  disabled = pythonOlder "3.6";
```

https://github.com/python-lsp/python-lsp-black/blob/master/setup.cfg#L18